# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../llvm/ccclib.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- llvm-2.5/utils/clang/tools/ccc/Makefile.vanilla	2009-03-18 15:34:08.000000000 +0100
+++ llvm-2.5/utils/clang/tools/ccc/Makefile	2009-03-18 15:34:14.000000000 +0100
@@ -18,7 +18,7 @@
 $(PROJ_bindir)/ccclib : $(Extra)
 	$(Echo) Installing ccclib.
 	$(Verb) mkdir -p "$(PROJ_bindir)/ccclib"
-	$(Verb) cp -p $? "$(PROJ_bindir)/ccclib"
+	$(Verb) cp -r $? "$(PROJ_bindir)/ccclib"
 	$(Verb) python -m compileall -d "$(PROJ_prefix)/bin/ccclib" "$(PROJ_bindir)/ccclib"
 	$(Verb) touch "$(PROJ_bindir)/ccclib"
 
